testing/common.sh

```diff
@@ -601,10 +601,10 @@ function install_operator_version() {
   if [ -z "$version" ]; then
     version=$(curl https://api.github.com/repos/minio/operator/releases/latest | jq --raw-output '.tag_name | "\(.[1:])"')
   fi
-  echo "Target operator release: $version"
+  echo "Target operator release: $version" # version value example is 5.0.14
```

Comment:
Shouldn't this be 5.0.15?

Reply:
Not really. Im not sure that we need to leave a hardcoded value in a comment but I suppose this is just an example. 

In this test and specific PR `deploy-tenant-upgrade.sh`, we are comparing the latest Operator release (5.0.14) to the current branch of Operator (master).